views/index.hbs

```diff
@@ -1,2 +1,16 @@
-<h1>{{title}}</h1>
-<p>Welcome to {{title}}</p>
+<div class="container-fluid d-flex flex-column ">
+    <div>
+        <nav class="navbar p-2">
+            <a href=""><h1>{{title}}</h1></a>
+            <a class="navbar-brand" href="#"><p>Welcome to {{title}}</p></a>
+        </nav>
+    </div>
+
+    <div class="p-2 d-flex justify-content-center">
+        
+             <button class="btn check-btn">
```

Comment:
Same remark as earlier, having `<a>` around `<button>` makes more sense.

<a href="/drones">
  <button class="btn check-btn">Check out the drones</button>
</a>